.github/workflows/react-app-lint.yml

```diff
@@ -0,0 +1,32 @@
+name: React App Linting Check
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    branches: [main]
+    paths:
+      - "react-app/**"
+      - ".github/workflows/react-app-lint.yml"
+  workflow_dispatch:
+
+jobs:
+  lint-api:
+    name: Lint React App
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      # Could optimize this by only installing necessary dependencies
+      - name: Install Dependencies
+        working-directory: ./react-app
+        run: npm i --only=dev
```

Comment:
I always assumed this worked, but now I see this output:
<img width="539" alt="image" src="https://github.com/bcgov/PIMS/assets/37922247/ff64ffce-511e-4a31-aba4-e106bf7066e4">

Doesn't hurt to have it in, but maybe it's not doing what we thought.

Reply:
After some searching it seems like this flag stopped working in 2020, any updated command seems to not replicate what this did.